kitsune/motidings/tests/__init__.py

```diff
@@ -0,0 +1,10 @@
+from factory.django import DjangoModelFactory
+from tidings.models import Watch
+
+
+class WatchFactory(DjangoModelFactory):
+    class Meta:
+        model = Watch
+
+    event_type = 'fooevent'
+    is_active = True
```

Comment:
I don't really have an opinion one way or the other. It won't change any of the rest of the code I wrote.

What do you want to do?